lib/utils.js

```diff
@@ -233,7 +233,34 @@ function unwrapEl (el) {
   return el;
 }
 
+async function clearLogs (locations, clear = true) {
```

Comment:
seems a little odd to me to have a function which takes as a boolean parameter whether or not that function is going to do anything. what about instead, up where we call this, doing:

```js
if (!!this.opts.clearLog) {
  await utils.clearLogs(LOG_LOCATIONS);
} else {
  log.info("Not clearing logfiles");
}
```